hivemind/optim/collaborative.py

```diff
@@ -208,7 +210,8 @@ def step(self, batch_size: Optional[int] = None, **kwargs):
             self.collaboration_state_updated.set()
             self.update_scheduler()
 
-            logger.log(self.status_loglevel, f"Optimizer step: done!")
+            logger.log(self.status_loglevel, f"Optimizer step: done! "
+                                             f"Your current contribution: {self.samples_processed} samples")
```

Comment:
I'd prefer to report the current contribution on the INFO level repeatedly regardless of the verbosity level, but for now, it's probably OK as is